scripts/test/runtests.ps1

```diff
@@ -5,6 +5,14 @@
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
+$failCount = 0
+
+# Capture PATH for later
```

Comment:
This should use the PATH from the outer scope. There's no need for this script to be aware of what it's testing. It's just testing whatever's on the path.